src/config/index.ts

```diff
@@ -139,19 +138,27 @@ export class Config {
     return partnerConfig.mutate && partnerConfig.mutate(payload, chtApi, isReplacement);
   }
 
-  public static getAuthenticationInfo(domain: string) : AuthenticationInfo {
-    const domainMatch = Config.getDomains().find(c => c.domain === domain);
+  public static getAuthenticationInfo(domain: string): AuthenticationInfo {
+    const availableDomains = Config.getDomains();
+
+    //if config has default config i.e a new install
+    //return passed domain for api config upload authentication
+    if (availableDomains.some(c => c.domain === 'localhost')) {
```

Comment:
I don't understand the comment here, why are you explicitly checking for localhost?

Reply:
updated